azext_edge/tests/edge/checks/test_mq_checks_unit.py

```diff
@@ -192,7 +210,12 @@ def test_check_mq_by_resource_types(ops_service, mocker, mock_resource_types, re
                     },
                     "mode": "distributed",
                 },
-                status={"runtimeStatus": {"status": ResourceState.running.value, "statusDescription": ""}},
+                status={
+                    "runtimeStatus": {
+                        "status": ResourceState.running.value,
+                        "statusDescription": "All replicas are running.",
```

Comment:
is this still supposed to be status description?

Reply:
updated!